sklearn/preprocessing/_data.py

```diff
@@ -1094,6 +1094,18 @@ def maxabs_scale(X, *, axis=0, copy=True):
     X_tr : {ndarray, sparse matrix} of shape (n_samples, n_features)
         The transformed data.
 
+    .. warning:: Risk of data leak
```

Comment:
Thanks for the PR! I think you don't want to include this change in this PR, though.